code/__defines/species_languages.dm

```diff
@@ -54,6 +54,7 @@
 #define LANGUAGE_TERMINATOR "Hephaestus Darkcomms"	// HKs.
 
 // Lesser-form Languages
+#define LANGUAGE_BRAZILIAN "Gibbering"				// alberyk
```

Comment:
First, I was told not to reference people, even other coders, in code comments.
Second, this just serves to create confusion. Name it `LANGUAGE_GIBBERING` or something that makes its purpose clear.